lib/rubocop/cop/rails/action_controller_testing.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Rails
+      # Using ActionController::TestCase is discouraged and should be replaced by
+      # ActionDispatch::IntegrationTest. Controller tests are too close to the
+      # internals of a controller whereas integration tests mimic the browser/user.
+      #
+      # @safety
+      #   This cop's autocorrection is unsafe because the API of each test case class is different.
+      #   Make sure to update each test of your controller test cases after changing the superclass.
+      #
+      # @example
+      #   # bad
+      #   class MyControllerTest < ActionController::TestCase
+      #   end
+      #
+      #   # good
+      #   class MyControllerTest < ActionDispatch::IntegrationTest
+      #   end
+      #
+      class ActionControllerTesting < Base
```

Comment:
Can you rename `ActionControllerTestCase` instead of `ActionControllerTesting`?

```suggestion
      class ActionControllerTestCase < Base
```